index/models.py

```diff
@@ -1,85 +1,100 @@
 from django.db import models
 
-class Petition(models.Model):
-	title = models.CharField(max_length=200)	#title of petition
-	description = models.CharField(max_length=4000)	#the paragraph description
-	ID = models.IntegerField(999999, primary_key=True)
-	archived = models.BooleanField(default=False)
-	hidden = models.BooleanField(default=False)
-				
-	created_date = models.DateTimeField(db_index=True, default=timezone.now)	#files the date created
-	expected_sig = models.IntegerField(300)		#the expected signature to move to the next step
-
-	author = models.ForeignKey(User, related_name='petitions')	#the author of the petition
-	tags = models.ManyToManyField(Tag, related_name='petitions')	#the tags, probably a max of 3
-	signatures = models.ManyToManyField(Signature, related_name='petitions')	#the signature
-
-	senate_response = models.ForeignKey(Responses, related_name='petitions')		#If the senate has responded , their answer
 
-	
-
-	def __unicode__(self):
-        return self.title		#returns title when asked for the item
+class Petition(models.Model):
+    # title: title of petition
```

Comment:
So after talking with @andrewdaoust and @garoller we were thinking it would be best to add a block comment about the summary of the class.
Ref: https://google.github.io/styleguide/pyguide.html#384-classes

Reply:
sounds good, I was planning on working on documentation this week anyways so I will make sure to change it so it better fits the style guides

Reply:
Yes, that would be a good change. I think that should be done in a new pull request though. This pull has a lot of other changes as well so I think it's important to merge it.

Reply:
Alright sounds good, we'll add those changes in another pull request

Reply:
I don't see any changes other than comments that got moved around, this doesn't need to get merged especially since it isn't following the block commenting styling.

Reply:
I agree with @andrewdaoust and @poulor, I think we should use the block commenting style referenced above. I also think we should combine this with changes to the model.

I'm going to close this PR for now, thanks.